src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -35,29 +30,33 @@
 import com.ericsson.ei.utils.ResponseMessage;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
 /**
- * This class represents the REST GET mechanism to extract the aggregated data on the basis of the SubscriptionName from the Missed Notification
- * Object.
+ * This class represents the REST GET mechanism to extract the aggregated data
+ * on the basis of the SubscriptionName from the Failed Notification Object.
```

Comment:
Can you possibly rephrase this sentence into something more human readable? How about: "This class contains logic for retrieving failed notifications for the given subscription."

Reply:
done

Reply:
Since this class was removed, update the other class "FailedNotificationControllerImpl" which is left :)

Reply:
But it is already removed...?